src/server_manager/web_app/app.ts

```diff
@@ -38,15 +41,6 @@ const CHANGE_HOSTNAME_VERSION = '1.2.0';
 export const DATA_LIMITS_AVAILABILITY_DATE = new Date('2020-06-02');
 const MAX_ACCESS_KEY_DATA_LIMIT_BYTES = 50 * (10 ** 9);  // 50GB
 
-interface UiAccessKey {
-  id: string;
-  placeholderName: string;
-  name: string;
-  accessUrl: string;
-  transferredBytes: number;
-  relativeTraffic: number;
-}
-
```

Comment:
While we're at it, do we also want to move `DisplayDataAmount` to JSDoc?

Reply:
Done